prow/cluster/private-arm/kubernetes_external_secrets.yaml

```diff
@@ -0,0 +1,6 @@
+# Empty
```

Comment:
nit: if its empty do we really need prow/cluster/private-arm/kubernetes-external-secrets_deployment.yaml etc?

Reply:
I guess so, maybe we do it later though